server/services/ItemsService.js

```diff
@@ -15,11 +15,11 @@ class ItemsService {
         return this.dbInstance.getById("items", id);
     }
 
-    getByArrayId(id) {
-        return this.dbInstance.getByArrayId("items", id);
+    getByIdArray(id) {
+        return this.dbInstance.getByIdArray("items", id);
     }
 
-    pagination(pageSize, pageNumber, items) {
+    pagination(pageSize = 4, pageNumber = 1, items) {
         if (pageSize) {
             this.pageSize = pageSize;
         }
```

Comment:
- you have added default value, you can now remove if statement then. 
- and also, I believe, it is a good practice, to put all parameters with default values in the end of the parameters list, so we could call `pagination(items)` and it will still work